### PR TITLE
HAL CHIBIOS: Make Solo's older cube a child of fmuv3 instead of CubeBlack

### PR DESCRIPTION
**Pull request by Pedals2Paddles**
Update the CubeBlack-Solo hardware def to use FMUv3 as its parent rather than CubeBlack.  Since this older cube doesn't have the same hardware as a new CubeBlack, it is causing conflicts.  Nearly everything else is the same.  Tested and working without becoming a brick.

### CODE REVIEW

**Comment by proficnc** [comment]
Maybe we should call it CubeSolo to reduce confusion?

File: libraries/AP_HAL_ChibiOS/hwdef/CubeBlack-solo/hwdef.dat [hunk 1]
```diff
@@ -3,7 +3,12 @@
 # this is based on the CubeBlack hwdef, with Solo's required parameter defaults
```
Comment (peterbarker) [inline libraries/AP_HAL_ChibiOS/hwdef/CubeBlack-solo/hwdef.dat]: Not it's not :-)

File: libraries/AP_HAL_ChibiOS/hwdef/CubeBlack-solo/hwdef.dat [hunk 2]
```diff
@@ -3,7 +3,12 @@
...
 # do not use this hwdef with any configuration other than a 3DR Solo with a stock or black cube running Copter 3.7.
```
Comment (peterbarker) [inline libraries/AP_HAL_ChibiOS/hwdef/CubeBlack-solo/hwdef.dat]: Also very bad advice.

File: libraries/AP_HAL_ChibiOS/hwdef/CubeBlack-solo/hwdef.dat [hunk 3]
```diff
@@ -3,7 +3,12 @@
...
-include ../CubeBlack/hwdef.dat
+include ../fmuv3/hwdef.dat
+
+env OPTIMIZE -O3
+
+SPIDEV icm20948_ext   SPI4 DEVID1  MPU_EXT_CS   MODE3  4*MHZ  8*MHZ
+SPIDEV icm20602_ext   SPI4 DEVID3  GYRO_EXT_CS  MODE3  4*MHZ  8*MHZ
 
 # pull Solo's default parameters from /Tools/Frame_params
 # these are parameters the Solo requires for proper operation that are differnet from the 3.7 standard defaults.
```
Comment (peterbarker) [inline libraries/AP_HAL_ChibiOS/hwdef/CubeBlack-solo/hwdef.dat]: speeling error while you're at it? :-)

**Comment by Pedals2Paddles** [comment]
Changed the hwdef to **CubeSolo** and updated the readme appropriately.

**Pull request merged**